### ISSUE TITLE
How to set custom image for user location icon

### ISSUE THREAD

**Issue by Cirff** [homeeondemand/react-native-mapbox-navigation#92]
![location_user](https://user-images.githubusercontent.com/107471523/173758891-d93c7857-6833-4171-a744-e15c74624fba.jpg)
how to replace above image with custom car image ??

**Comment by ahmedelsayed11** [comment]
Has android version worked with you ? @Cirff

**Comment by Cirff** [comment]
yes in andriod it is working. but i need to change this icon.

**Comment by ahmedelsayed11** [comment]
Yes it's working on Android I have fixed the problem but how we can change some custom components for the library or how we can add our own edit to native iOS or android ?
